Sources/Validation/Convenience/Email.swift

```diff
@@ -3,15 +3,12 @@ import Foundation
 public struct EmailValidator: Validator {
 
     public init() {}
-	
-	private static let name = "[a-z0-9!#$%&'*+/=?^_`{|}~]"
```

Comment:
Were these special characters not technically correct? I've seen people with special characters in their email such as: "ç", "å", "µ", "∆". think of Greece, Sweden, Spain, Russia and some of the arabic speaking countries

Reply:
even as it was previously the characters you put would be marked incorrect. maybe we should look at what other frameworks do? the only issue seems to be the length of the TLD.

Reply:
We could invert this. Accept all characters _except_ `@`, `.`, ...